decision_service/ds.sln

```diff
@@ -0,0 +1,38 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 14
+VisualStudioVersion = 14.0.25420.1
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "dsclientlib", "dsclientlib\dsclientlib.vcxproj", "{C40E3263-B6FF-46D5-92F7-D67E6F9FCEDD}"
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "unit_test", "unit_test\unit_test.vcxproj", "{18A3556E-493E-4DD5-8C33-760185B764D4}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|x64 = Debug|x64
+		Debug|x86 = Debug|x86
+		Release|x64 = Release|x64
+		Release|x86 = Release|x86
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{C40E3263-B6FF-46D5-92F7-D67E6F9FCEDD}.Debug|x64.ActiveCfg = Debug|x64
+		{C40E3263-B6FF-46D5-92F7-D67E6F9FCEDD}.Debug|x64.Build.0 = Debug|x64
+		{C40E3263-B6FF-46D5-92F7-D67E6F9FCEDD}.Debug|x86.ActiveCfg = Debug|Win32
+		{C40E3263-B6FF-46D5-92F7-D67E6F9FCEDD}.Debug|x86.Build.0 = Debug|Win32
```

Comment:
>|x86.Build.0 = Debug|Win32 [](start = 46, length = 26)

it's supported?

Reply:
Ran all unittests in x86, Release and Debug.  It seems to work.  Leaving it in till I hit a x86 implementation incompatibility.

---
In reply to: [186733028](https://github.com/JohnLangford/vowpal_wabbit/pull/1479#discussion_r186733028) [](ancestors = 186733028)